src/main/java/org/mongounit/MongoUnitUtil.java

```diff
@@ -261,13 +260,9 @@ private static List<String> getCollectionNamesToUse(
    */
   private static List<String> getCollectionNames(MongoDatabase mongoDatabase) {
 
-    List<String> collectionNames = new ArrayList<>();
-
     // Retrieve collection names from db
-    for (String collectionName : mongoDatabase.listCollectionNames()) {
-      collectionNames.add(collectionName);
-    }
-
+    List<String> collectionNames = new ArrayList<>();
+    mongoDatabase.listCollectionNames().forEach(collectionNames::add);
```

Comment:
@alexisGandar Same question here... This update is just to use the newer syntax in Java, correct? I don't think this code was breaking with the version update, was it?

Reply:
Same comment thant above => It's not related to the java version. It's related to the upgrade of MongoDB drivers. This is the reason of this PR. After upgrading spring boot to 3.3.0 the MongoDB driver sub-dependency were also updated and in the update they changed the return type of the listCollectionNames. This change was required to avoid an error.

Reply:
Strange that you'd get an error since `public interface ListCollectionNamesIterable extends MongoIterable<String>` so it extends `MongoIterable<String>`.

Either way, it's update to the syntax, so whatever. Merging.